app/Migrations/Version20240306114323.php

```diff
@@ -22,7 +22,6 @@ public function up(Schema $schema): void
         $table->setPrimaryKey(['id']);
         $table->addIndex(['item_id']);
         $table->addIndex(['owner_id']);
-        $table->addUniqueIndex(['item_id', 'owner_id']);
```

Comment:
This will require some manual work then to remove existing unique indexes?

Otherwise a new migration removing if exists would be better no?

Reply:
True!

But the issue is that the unique index was created without a name, this name will be random so it can't be deleted with a migration. I already discussed this with @willaerk on Slack and already did the manual removal of these indexes. The tables on TEST and PROD were still empty.

Also, the acceptance tests start from scratch and they still work.